test/Main.hs

```diff
@@ -82,6 +83,7 @@ main = do
     , ("Feature.DeleteSpec"       , Feature.DeleteSpec.spec)
     , ("Feature.InsertSpec"       , Feature.InsertSpec.spec)
     , ("Feature.QuerySpec"        , Feature.QuerySpec.spec)
+    , ("Feature.RpcSpec"          , Feature.RpcSpec.spec)
```

Comment:
Oops, I was on a small screen and it was just the line wrapping on github's preview that confused me.